ui/appui-react/src/appui-react/childwindow/InternalChildWindowManager.tsx

```diff
@@ -326,28 +326,19 @@ export class InternalChildWindowManager implements FrameworkChildWindows {
     childWindow.shouldUseOuterSized =
       navigator.userAgent.toLowerCase().indexOf("edg/") > -1;
 
-    if (0 === url.length) {
-      childWindow.document.write(childHtml);
-      this.renderChildWindowContents(
-        childWindow,
-        childWindowId,
-        content,
-        title
-      );
-    } else {
-      childWindow.addEventListener(
-        "load",
-        () => {
-          this.renderChildWindowContents(
-            childWindow,
-            childWindowId,
-            content,
-            title
-          );
-        },
-        false
-      );
-    }
+    childWindow.addEventListener(
+      "DOMContentLoaded",
+      () => {
+        childWindow.document.write(childHtml);
```

Comment:
But why would we write a fully defined HTML (page structure) when we load one from `iTwinPopup.html`? Is that over-riden or appended?

Reply:
Actually, you're right about this, my bad. I got confused why we we have two different ways to load page structure. Seems like the `iTwinPopup.html` is only used in test-apps but I don't see a reason why we need to load structure differently in test-apps. Maybe we should just remove that and always use `childWindow.document.write(childHtml)`?

Reply:
HTML page might be a bit more configurable, let's keep it if we can.

Reply:
Ok but I'm worried about a situation where users would pass `useDefaultPopoutUrl` as `true` or set `UiFramework.useDefaultPopoutUrl` to `true` (which means the same thing) and then it will try to use that `iTwinPopup.html` but throw an error because it does not exist. This html file is internal for test-apps.

Reply:
It is documented what the options does - application is responsible to have that route setup.

Reply:
Alright, I added the conditional write back. Also changed the empty url string to `popout` because empty url does not seem to emit load events.

Reply:
I'm not too sure sure about that `popout` change :? https://developer.mozilla.org/en-US/docs/Web/API/Window/open

Reply:
This is not relevant anymore. Resolving